packages/orchestration/src/exos/cosmos-orchestration-account.js

```diff
@@ -572,6 +603,60 @@ export const prepareCosmosOrchestrationAccountKit = (
           return coerceCoin(chainHub, amount);
         },
       },
+
+      updateTxProgressWatcher: {
+        /**
+         * @param {readonly [
+         *   agoric: ChainInfo<'cosmos'>,
+         *   la: LocalIbcAddress,
+         *   ra: RemoteIbcAddress,
+         * ]} param0
+         * @param {{
+         *   progressTracker: ProgressTracker;
+         *   trafficEntryIndex: number;
+         *   protocol: 'ibc';
+         * }} opts
+         */
+        onFulfilled(
+          [agoric, la, ra],
+          { progressTracker, trafficEntryIndex, protocol },
+        ) {
+          protocol;
+          const lad = decodeIbcEndpoint(la);
+          const rad = decodeIbcEndpoint(ra);
+
+          const priorReport = progressTracker.getCurrentProgressReport();
+          const report = {
+            ...priorReport,
+            traffic: priorReport.traffic?.map(
```

Comment:
for readability please pull up `const traffic =…`

This this can be more concise and harden in the right place:
```
          const report = harden({
            ...priorReport,
            traffic,
          });
```

Reply:
:+1: